examples/README

```diff
@@ -1 +1,84 @@
-Here are two sample programs which use QTermWidget for displaying a terminal
+Examples using QTermWidget
+==========================
+
+This directory contains minimal examples showing how to embed and use `QTermWidget` in both C++ and Python (PyQt/PySide) applications.
+
+Contents
+--------
+* `cpp/main.cpp` – Basic single–terminal C++ example with menu actions (Find / Copy / Paste / About Qt) and runtime selection of color schemes and key bindings via command‑line arguments.
+* `cpp/RemoteTerm/` – A simple remote client example that connects to a TCP server exposing a pseudo terminal. Local keystrokes are sent to the remote host; data received over the socket is written into the local PTY backing the widget.
+* `pyqt/` – Python examples (PyQt/PySide) demonstrating how to instantiate and configure the terminal widget from Python. See the files inside for details.
+
+Building C++ Examples
+---------------------
+
+Configure the build (top level) and enable example targets as needed:
+
+```
+mkdir -p build
+cd build
+cmake -DBUILD_EXAMPLE=ON -DBUILD_EXAMPLE_REMOTETERM=ON ..
+cmake --build . -j
```

Comment:
Personally I'm not a fan of `-j`, which often makes the system hang, especially on not-so-powerful machines.